### ISSUE TITLE
Erroneous request to '/undefined' when no sources are provided for Avatar

### ISSUE THREAD

**Issue by jamesbirtles** [vnphanquang/svelte-put#151]
When no sources are provided for the Avatar component an attempt to load an image at `/undefined` (or whatever you path is + `/undefined`) is made.

If there ends up being no sources then it should probably just go straight to using the default fallback, rather than waiting for an error on loading `/undefined`.

I think changing this line 
https://github.com/vnphanquang/svelte-put/blob/f1293f367c9046d9b98aeed47b35347a704f5f35/packages/misc/avatar/src/lib/Avatar.svelte#L40

to something like
```js
rElement.src = rSrc = sources[currentSourceIndex] ?? DEFINITIVE_FALLBACK;
```
is all thats needed.

**Comment by vnphanquang** [comment]
Ah, nice catch @jamesbirtles. Thanks for the input. I will look into this.

**Comment by vnphanquang** [comment]
patch released with [@svelte-put/avatar@2.0.1](https://github.com/vnphanquang/svelte-put/pull/164). Thanks again for reporting.
